src/main/java/com/iridium/iridiumskyblock/utils/Placeholder.java

```diff
@@ -18,7 +18,7 @@
      * @param value The actual value of the placeholder
      */
     public Placeholder(String key, String value) {
-        this.key = "{" + key + "}";
+        this.key = "%" + key + "%";
```

Comment:
Need to change the placeholders in inventories.json and other configs to match this